src/exojax/spec/api.py

```diff
@@ -0,0 +1,636 @@
+        self.gpp = jnp.array(self.gpp)
+        self.jlower = jnp.array(self.jlower, dtype=int)
+        self.jupper = jnp.array(self.jupper, dtype=int)
+        self.alpha_ref = jnp.array(self.alpha_ref)
+        self.n_Texp = jnp.array(self.n_Texp)
+        
+    def QT_interp(self, T):
+        """interpolated partition function.
+
+        Args:
+           T: temperature
+
+        Returns:
+           Q(T) interpolated in jnp.array
+        """
+        return jnp.interp(T, self.T_gQT, self.gQT)
+
+    def qr_interp(self, T):
+        """interpolated partition function ratio.
+
+        Args:
+           T: temperature
+
+        Returns:
+           qr(T)=Q(T)/Q(Tref) interpolated in jnp.array
+        """
+        return self.QT_interp(T) / self.QT_interp(Tref)
+
+
+class MdbHitemp(HITEMPDatabaseManager):
+    """molecular database of HITEMP.
+
+    Attributes:
+        simple_molecule_name: simple molecule name
+        nurange: nu range [min,max] (cm-1)
+        nu_lines (nd array): line center (cm-1)
+        Sij0 (nd array): line strength at T=Tref (cm)
+        dev_nu_lines (jnp array): line center in device (cm-1)
+        logsij0 (jnp array): log line strength at T=Tref
+        A (jnp array): Einstein A coeeficient
+        gamma_natural (jnp array): gamma factor of the natural broadening
+        gamma_air (jnp array): gamma factor of air pressure broadening
+        gamma_self (jnp array): gamma factor of self pressure broadening
+        elower (jnp array): the lower state energy (cm-1)
+        gpp (jnp array): statistical weight
+        n_air (jnp array): air temperature exponent
+    """
+    def __init__(self,
+                 path,
+                 nurange=[-np.inf, np.inf],
+                 margin=0.0,
+                 crit=0.,
+                 Ttyp=1000.,
+                 isotope=None,
+                 gpu_transfer=False):
+        """Molecular database for HITRAN/HITEMP form.
+
+        Args:
+           path: path for HITEMP par file
+           nurange: wavenumber range list (cm-1) [min,max] or wavenumber grid
+           margin: margin for nurange (cm-1)
+           crit: line strength lower limit for extraction
+           Ttyp: typical temperature to calculate Sij(T) used in crit
+           isotope: None= use all isotopes. 
+           gpu_transfer: tranfer data to jnp.array?
+        """
+
+        self.path = pathlib.Path(path).expanduser()
+        self.molecid = search_molecid(str(self.path.stem))
+        self.simple_molecule_name = get_molecule(self.molecid)
+        self.crit = crit
+        self.Ttyp = Ttyp
+        self.margin = margin
+        self.nurange = [np.min(nurange), np.max(nurange)]
+        load_wavenum_min = self.nurange[0] - self.margin
+        load_wavenum_max = self.nurange[1] + self.margin
+
+        super().__init__(
+            molecule=self.simple_molecule_name,
+            name="HITEMP-{molecule}",
+            local_databases=self.path.parent,
```

Comment:
Rewriting this line with 
```
local_databases="/",
```
looks resolve the above error for `MdbHitemp`. 
Is it ok?